### ISSUE TITLE
Accept cidr notation for ip.route.network specification

### ISSUE THREAD

**Issue by tyll** [linux-system-roles/network#30]
Currently it accepts only an IP address and requires the prefix to contain the router mask.

**Comment by pcahyna** [comment]
I propose that it be done only after there are tests for the functionality that will be touched by implementing this.

**Comment by tyll** [comment]
The good thing is that this only affects the parser, therefore it is very likely already covered by the unit tests. I will make sure that it is before changing something there.

**Comment by tyll** [comment]
there have been no requests for a long time, so I close this.
